src/main/webapp/resources/js/components/samples/components/SampleFileContenate.tsx

```diff
@@ -99,15 +109,19 @@ export function SampleFileConcatenate({
           }
 
           dispatch(resetConcatenateSelected());
-          dispatch(addToSequenceFiles({ sequenceFiles: data }));
+          dispatch(
+            addToSequenceFiles({
+              sequenceFiles: sampleSequencingObjectFileModels,
+            })
+          );
           notification.success({ message });
           form.resetFields();
           setVisible(false);
+          setConcatenating(false);
         })
         .catch((error) => {
-          notification.error({
```

Comment:
You can probably even move the notification into the `transformResponse`

Reply:
I think the notifications are better left here. We aren't transforming the response format or anything

Reply:
Sounds good.  The only reason I put them there is so that wherever this is called from, the notification gets displayed.  There is probably a better way to handle that then in `transfromResponse`